lib/Backend/IRBuilderAsmJs.cpp

```diff
@@ -2921,6 +2921,14 @@ IRBuilderAsmJs::BuildFloat3(Js::OpCodeAsmJs newOpcode, uint32 offset, Js::RegSlo
         instr = IR::Instr::New(Js::OpCode::Copysign_A, dstOpnd, src1Opnd, src2Opnd, m_func);
         break;
 
+    case Js::OpCodeAsmJs::Min_Flt:
+        instr = IR::Instr::New(Js::OpCode::MinNaN, dstOpnd, src1Opnd, src2Opnd, m_func);
```

Comment:
I don't really like calling this MinNaN. IMO would rather call this something like WasmMin, or not create new opcode and in lowering check to see if the function is a wasm function